src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java

```diff
@@ -89,7 +86,6 @@ private void uploadNewDocument(File file, CreateTemplateRenditionDto createTempl
             }
 
         } catch (RuntimeException | IOException e) {
-            log.error(e.getMessage(), e);
```

Comment:
may be better to keep with `log.error("{description }", e)';

Reply:
SonarQube complains about that (Exceptions should be either logged or rethrown but not both - java:S2139).

Reply:
strange we hame many like this logging and throwing another exception.

Reply:
maybe java:S2139 is suppressed in those places?